.github/ISSUE_TEMPLATE/report-bug.md

```diff
@@ -0,0 +1,42 @@
+---
+name: "🐞 Report a Bug"
+about: Report a bug encountered while using the Portmaster
+labels: bug
+---
+
+<!--
+Please disclose security related issues privately to support@safing.io.
+-->
+
+**Pre-Submit Checklist**:
+
+- Check known issues:
+  - [Windows](https://docs.safing.io/portmaster/install/windows#known-issues)
+  - [Linux](https://docs.safing.io/portmaster/install/linux#compatibility)
+- Search for similar/existing issues on Github.
```

Comment:
I am unsure if we should move away from having checklists to fill out - ppl seemed to do this well until now. What is your take?